ToDo-Garden/ToDo-Garden/TimerScene/Package.swift

```diff
@@ -0,0 +1,39 @@
+// swift-tools-version: 5.10
+import PackageDescription
+
+let package = Package(
+  name: "TimerScene",
+  platforms: [.iOS(.v15)],
+  products: [
+    .library(
+      name: "TimerScene",
+      targets: ["TimerScene"]
+    )
+  ],
+  dependencies: [
+    .package(path: "../ToDoGardenUI")
+  ],
+  targets: [
+    .target(
+      name: "TimerScene",
+      dependencies: [
+        "TimerSceneEntity",
+        "TimerSceneApi",
+        .product(name: "ToDoGardenUIComponent", package: "ToDoGardenUI")
+      ]
+    ),
+    .target(
+      name: "TimerSceneApi",
```

Comment:
`p1`

```suggestion
      name: "TimerSceneAPI",
```
다른 패키지와의 일관성을 위해 `TimerSceneAPI` 이어야 할 것 같습니다 :)